scripts/export_live_research_records.py

```
@@ -468,20 +468,14 @@ def _lookup_provider_sort_strategy_full(
 
     ``declared_sort``       – strategy explicitly configured for *provider_key*
                               in the protocol (empty string when absent).
-    ``applied_sort``        – strategy that will actually be used; may be
-                              inferred from a sibling provider for OpenAlex.
+    ``applied_sort``        – strategy that will actually be used; equals
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)